gdpr_assist/models.py

```diff
@@ -181,16 +192,18 @@ def get_export_filename(self):
             self.model._meta.object_name,
         )
 
-
 class PrivacyModel(models.Model):
     """
     An abstract model base class with support for anonymising data
     """
     anonymised = models.BooleanField(default=False)
+    retention_policy = models.ForeignKey(to="gdpr_assist.RetentionPolicyItem", on_delete=models.SET_NULL,
+                                         related_name="target_%(app_label)s_%(class)s", null=True, blank=True)
```

Comment:
my first attempt was to just make the `related_name` to be `target_entities`. But, that clashed because each model that subclassed `PrivacyModel` used that. So, using `policy.target_entites` was ambiguous and python complained. 

I assume the same would happen if you left it and it just went to an ambiguous `retentionpolicy_set` like you suggest above.

This way, we get 

- `target_seminar_users`
- `target_poll_pollchoices`
....

and then we use some magic methods to combine them all into `target_entities` so the user doesnt have to worry about types.